libs/src/services/discoveryProvider/index.js

```diff
@@ -528,7 +531,7 @@ class DiscoveryProvider {
   }
 
   /**
-   * Gets the healthy discovery provider endpoint used in creating the axious request later.
+   * Gets the healthy discovery provider endpoint used in creating the axios request later.
    * If the number of retries is over the max count for retires, clear the cache and reselect
    * another healthy discovery provider. Else, return the current discovery provider endpoint
    * @param {int} attemptedRetries the number of attempted requests made to the current disc prov endpoint
```

Comment:
nit, in js this should be @param {number} (I know you didn't make this change just now)

Reply:
:3